### PR TITLE
modify genesis-base version, add hwdb for lspci and udevadm

### PR DESCRIPTION
**Pull request by zet809**

### CODE REVIEW

**Comment by zet809** [comment]
This pull request also include the modifications for building genesis-base on Fedora 23 ppc64.

File: xCAT-genesis-builder/install [hunk 1]
```diff
@@ -11,7 +11,7 @@ dracut_install mkswap df brctl vconfig ifenslave ssh-keygen scp clear dhclient l
 dracut_install lldptool /lib64/libnss_dns-2.12.so /lib64/libnss_dns.so.2
 dracut_install poweroff ntpq ntpd hwclock date /usr/share/terminfo/x/xterm /usr/share/terminfo/s/screen /etc/nsswitch.conf /etc/services
 dracut_install /sbin/rsyslogd /etc/protocols umount /bin/rpm /usr/lib/rpm/rpmrc
-dracut_install chmod /lib/libc.so.6 /lib/ld-linux.so.2 /lib/libdl.so.2 /lib/libm.so.6 /sbin/route /sbin/ifconfig /usr/bin/whoami /usr/bin/head /usr/bin/tail basename /etc/redhat-release ping tr lsusb /usr/share/hwdata/usb.ids #ibm fw wrapper requirements
```
Comment (hu-weihua) [inline xCAT-genesis-builder/install]: I think the step is right, but it isn't convenience to review or to maintain later, so could you put the step which support ``lspci`` and ``udevadm`` database into a new line and add some description in comments.

File: xCAT-genesis-builder/xCAT-genesis-base.spec [hunk 2]
```diff
@@ -13,6 +13,8 @@ BuildArch: noarch
 %define __spec_install_post :
 %define debug_package %{nil}
 %define __prelink_undo_cmd %{nil}
+# To fix the issue error: Arch dependent binaries in noarch package, the following line is needed on Fedora 23 ppc64
+%define _binaries_in_noarch_packages_terminate_build   0
```
Comment (hu-weihua) [inline xCAT-genesis-builder/xCAT-genesis-base.spec]: the build code is common for Power and X86_64, Are you sure it's fine in X86_64 environment? have we verified it in x86_64?

Comment (zet809) [inline xCAT-genesis-builder/xCAT-genesis-base.spec]: 1. system X is out of strategy. 2. no genesis-base build requirement we can get. So, I think we can keep it here and verify it until there is requirement to build xcat-genesis-base on x86_64.

File: xCAT-genesis-builder/install (hunk 1 above)
Comment (zet809) [inline xCAT-genesis-builder/install]: Modified, pls review.

File: xCAT-genesis-builder/xCAT-genesis-base.spec (hunk 2 above)
Comment (hu-weihua) [inline xCAT-genesis-builder/xCAT-genesis-base.spec]: Ok. I think that's fine.

**Comment by hu-weihua** [comment]
I have reviewed. I agree to merge.

**Pull request merged**